Tools/Pipeline/Templates/DXEffect.fx

```diff
@@ -0,0 +1,39 @@
+﻿matrix World;
+matrix View;
+matrix Projection;
+
+struct VertexShaderInput
+{
+	float4 Position : SV_POSITION;
+	float4 Color : COLOR0;
+};
+
+struct VertexShaderOutput
+{
+	float4 Position : SV_POSITION;
+	float4 Color : COLOR0;
+};
+
+VertexShaderOutput MainVS(in VertexShaderInput input)
+{
+	VertexShaderOutput output = (VertexShaderOutput)0;
+	output.Position = mul(input.Position, World);
+	output.Position = mul(output.Position, View);
+	output.Position = mul(output.Position, Projection);
```

Comment:
We should probably use `WorldViewProjection` as a single matrix here.  We want to give people a basic example, but also we don't want to lead them down the road of bad habbits.  You can explain in the comment the performance benefit of pre-combining the world, view, and projection on the CPU and not doing the same multiplication on every vertex.